pkg/collector/py/check_test.go

```diff
@@ -122,6 +122,12 @@ func TestInterval(t *testing.T) {
 	assert.Equal(t, time.Duration(1)*time.Second, c.Interval())
 }
 
+func TestExtraID(t *testing.T) {
+	c, _ := getCheckInstance("testcheck", "TestCheck")
+	c.Configure([]byte("namespace: test"), []byte("foo: bar"))
+	assert.Equal(t, string(c.ID()), "testcheck:test:d5c9af2b64a6eedd")
```

Comment:
Same here, let's test we're giving a correct output when no namespace field is set